src/plugins/DeveloperTools.ts

```diff
@@ -193,15 +178,25 @@ if ('__DEV__') {
     }
   };
 } else {
+  // tslint:disable-next-line: max-classes-per-file
+  DeveloperTools = class extends GenericPlugin {
+    public debugOverlay?: HTMLElement;
+
+    public onCreate() {
+      console.warn(`
+        %c [ADJS] ATTENTION! DeveloperTools module has been detected.
+        This plugin is meant to be used in development only. If you wish to enable for your current session,
+        in your console type 'AD_JS.DEBUG()'.
+      `, 'font-weight: bold;');
+    }
+  };
+
   (window as any)[`AD_JS`] = {
-    DEBUG: () => {
-      /*
-        TODO:
-        Since there is no server that we can just fetch
-        the developer bundle from, we need to decide how to get
-        new code to overwrite existing bundle.
-      */
-      return 'This feature is coming soon!. Please use the development build for the time being.';
+    DEBUG: async () => {
```

Comment:
This should be on the main adjs object. A user shouldn’t need to include any package in order to get the DEBUG ability